CardinalityEstimation.Test/CardinalityEstimatorTests.cs

```diff
@@ -65,6 +65,25 @@ public void TestGetSigma()
         }
```

Comment:
Note that the new property is not serialized by the custom serializer. However, that's fine - I'm planning to merge the Murmur3 branch soon, and it includes a breaking change in the serializer.  So go ahead and ignore the serializer for now.